test/test_unary_ufuncs.py

```diff
@@ -433,7 +433,7 @@ def compare_out(op, input, out):
             with self.assertRaises(RuntimeError):
                 op(input, out=output)
 
-    @ops(unary_ufuncs)
+    @ops(unary_ufuncs, dtypes=OpDTypes.supported)
```

Comment:
Is this only a forward-looking change since no ops define their "default/basic" test dtypes yet?